### PR TITLE
Uploading solved today's tasks

### PR DESCRIPTION
**Pull request by Pozharitskiy**

### CODE REVIEW

File: first.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+function summ() {
+    let currentSum = 0;
+    for (let i = 0; i < arguments.length - 1; i++) {
+        currentSum += +arguments[i];
```
Comment (kobweb) [inline first.js]: It would not be working if `'abc'` value will come not at the end.

File: second.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+function summAdvanced() {
+    let currentSum = 0;
+    for (let i = 0; i < arguments.length; i++) {
+        console.log(arguments[i]);
+        if (!isNaN(+arguments[i])) {
```
Comment (kobweb) [inline second.js]: You can make your logic more simple and clear, for example:
```
const arg = arguments[i];
const isFunction = typeof(arg) === 'function';
const argValue = isFunction ? arg() : arg;
const argNumberValue = Number(argValue);

if (argNumberValue) {
  sum += argNumberValue;
}
```

File: first.js (hunk 1 above)
Comment (Pozharitskiy) [reply first.js]: Fixed

File: second.js (hunk 2 above)
Comment (Pozharitskiy) [reply second.js]: Done. It's awesome you wrote!

**Pull request merged**